.github/workflows/build.yml

```diff
@@ -90,16 +90,16 @@ jobs:
     strategy:
       matrix:
         build_type: [Debug, Release]
-        os: [macos-latest, ubuntu-20.04]
+        os: [macos-latest, ubuntu-latest]
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2
       - name: Install Linux dependencies
         if: startsWith(matrix.os, 'ubuntu')
-        run: sudo apt-get install -y protobuf-compiler libprotobuf-dev libprotoc-dev
+        run: sudo apt-get install -y protobuf-compiler libprotobuf-dev libprotoc-dev libabsl-dev
```

Comment:
Why do we need `libabsl-dev` on Ubuntu? The absolute newest version of protobuf in the Debian/Ubuntu ecosystem is currently protobuf 3.21.12 in Debian unstable which pre-dates the absl dependency in newer versions of protobuf.

Reply:
That's right ... removed